GivenTwoArraysAreEqualOrNot.dart

```diff
@@ -0,0 +1,10 @@
+void main() {
+  List arr1 = [1, 2, 3];
+  List arr2 = [1, 2, 3];
+
+  if (arr1 == arr2) {
```

Comment:
the logic is not correct you are comparing the variable reference. you have to compare the element of the arrays